mongo/integration/sessions_test.go

```diff
@@ -205,71 +285,91 @@ func TestSessions(t *testing.T) {
 			})
 		}
 	})
-	mt.RunOpts("ended session", noClientOpts, func(mt *mtest.T) {
-		// an ended session cannot be used in commands
 
-		sessionFunctions := createFunctionsSlice()
-		for _, sf := range sessionFunctions {
-			mt.Run(sf.name, func(mt *mtest.T) {
-				sess, err := mt.Client.StartSession()
-				assert.Nil(mt, err, "StartSession error: %v", err)
-				sess.EndSession(context.Background())
+	mt.RunOpts("6 no further operations can be performed using a session after endSession has been called",
+		noClientOpts, func(mt *mtest.T) {
+			// an ended session cannot be used in commands
```

Comment:
Optional: Wrapping the subtest function to the next line makes the indentation kinda wonky. Consider defining the description separately to clean up the formatting.

E.g.
```go
const proseTest6 = "6 no further operations can be performed using a session after endSession has been called"
mt.RunOpts(proseTest6, noClientOpts, func(mt *mtest.T) {
	// ...
```